src/components/Layout/Page.tsx

```diff
@@ -36,7 +36,10 @@ const PageMeta = () => {
 
   const pageMeta = getCustomMeta(pathname, t) || {}
   const { title, description, image } = { ...DEFAULT_META, ...pageMeta }
-  const pageTitle = cakePriceUsdDisplay ? [title, cakePriceUsdDisplay].join(' - ') : title
+  let pageTitle = cakePriceUsdDisplay ? [title, cakePriceUsdDisplay].join(' - ') : title
+  if (symbol) {
```

Comment:
Update to existing code:
Showing symbol in the title of the page (e.g. if you'r on WBNB / BUSD analytics page it will show `WBNB / BUSD - PancakeSwap`